engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/util/TypeToDisplayTextNode.java

```diff
@@ -52,11 +50,12 @@ String doDisplay(
     } else if (TypesGen.isFunction(value)) {
       return "Function";
     } else if (value instanceof Atom atom) {
-      var cons = atom.getConstructor();
-      if (cons.getName().equals("Value")) {
-        return cons.getType().getName() + "." + cons.getName();
+      var constructor = atom.getConstructor();
+      var consName = constructor.getName();
+      if (consName.equals("Value") || consName.equals("Error")) {
```

Comment:
Shoud we maybe push this into Constructor? This starts to look like some hardcoded magic